pkg/handlers/authorization/handler.go

```diff
@@ -79,6 +80,7 @@ func (auth *authorizationHandler) Process(req *http.Request, context *handlers.R
 			}
 		}
 		req.Header.Add(headerForwardedRoles, strings.Join(context.RoleSet().List(), ","))
+		req.Header.Add(headerForwardedFor, "127.0.0.1")
```

Comment:
what about for ipv6 clusters?

Reply:
yes, that was my doubt too. can we know ipv4 or ipv6 and add `127.0.0.1` or `::1` ?

Reply:
As per [configuration](https://github.com/openshift/origin-aggregated-logging/blob/master/elasticsearch/sgconfig/config.yml#L7) of opendistro, the value we pass here will be pattern matched against `.*` , so either value is ok

Reply:
https://github.com/opendistro-for-elasticsearch/security/blob/b69e821a360a774ec2e94dc01390cd61521562d2/src/main/java/com/amazon/opendistroforelasticsearch/security/http/RemoteIpDetector.java#L219
and
https://github.com/opendistro-for-elasticsearch/security/blob/b69e821a360a774ec2e94dc01390cd61521562d2/src/main/java/com/amazon/opendistroforelasticsearch/security/http/XFFResolver.java#L103

Reply:
@vimalk78 right, but we eventually want to lock that down i think..
@jcantrill thoughts?

Reply:
updated

Reply:
we do need to lock it down otherwise you can access ES by going directly to it and spoofing who you are with the headers.  The configuration should support localhost for both ipv4 and 6 which I believe has to be some form of IP and not name

Reply:
The value sent now is `localhost`